samples/bluetooth/ibeacon/README.rst

```diff
@@ -0,0 +1,43 @@
+.. _bluetooth-ibeacon-sample:
+
+Bluetooth: iBeacon
+##################
+
+Overview
+********
+
+A simple application demonstrating the BLE Broadcaster role
```

Comment:
Let's make this a sentence:

    This simple application demonstrates the BLE ...